src/Monolog/Handler/SqsHandler.php

```diff
@@ -51,8 +52,8 @@ protected function write(array $record): void
         }
 
         $messageBody = $record['formatted'];
-        if (strlen($messageBody) >= static::MAX_MESSAGE_SIZE) {
-            $messageBody = substr($messageBody, 0, static::HEAD_MESSAGE_SIZE);
+        if (Utils::strlen($messageBody) >= static::MAX_MESSAGE_SIZE) {
+            $messageBody = Utils::substr($messageBody, 0, static::HEAD_MESSAGE_SIZE);
```

Comment:
Also should be undone

Reply:
Utils::substr work with bytes count now
Utils::strlen - removed